app/helpers/insured/families_helper.rb

```diff
@@ -419,7 +419,7 @@ def all_transitions(enrollment)
                              l10n('enrollment.latest_transition_data',
                                   from_state: transition.from_state,
                                   to_state: transition.to_state,
-                                  created_at: transition.created_at.in_time_zone('Eastern Time (US & Canada)').strftime("%m/%d/%Y %-I:%M%p"))
+                                  created_at: transition.created_at&.in_time_zone('Eastern Time (US & Canada)')&.strftime("%m/%d/%Y %-I:%M%p"))
```

Comment:
@saipraveen18 Can we just display the text for the information that is not available instead of safe navigation? With safe navigation, we are hiding the data issue. Thoughts?